integration/containerd/cri/integration-tests.sh

```diff
@@ -5,6 +5,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+[[ "${DEBUG}" != "" ]] && set -o xtrace
```

Comment:
Nit: This could be simplified to:

```suggestion
[ -n "${DEBUG}" ] && set -o xtrace
```